Detectors/TPC/simulation/src/Detector.cxx

```diff
@@ -205,6 +205,15 @@ Bool_t  Detector::ProcessHits(FairVolume* vol)
   const double rnd = refMC->GetRandom()->Rndm();
   refMC->SetMaxStep(0.2+(2.*rnd-1.)*0.05);  // 2 mm +- rndm*0.5mm step
   
+  // ===| check active sector |=================================================
+  //
+  // Get the sector ID and check if the sector is active
+  static TLorentzVector position;
+  refMC->TrackPosition(position);
+  const int sectorID = static_cast<int>(Sector::ToSector(position.X(), position.Y(), position.Z()));
+  // TODO: Temporary hack to process only one sector
+  if (sectorID != 0) return kFALSE;
```

Comment:
should this be put into the production version??

Reply:
Thanks for spotting this. I'll comment it out. We should discuss a more proper solution for this, e.g. an array over all sectors indicating the active state which can be set in the macro...